### PR TITLE
Add lhcb-simpletools

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready to be reviewed/merged

**Comment by chrisburr** [comment]
Thanks @xhochy ⚡️

File: recipes/lhcb-simpletools/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    # Fix up CMake for using conda's sysroot
+    # See https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_TOOLCHAIN_FILE=${RECIPE_DIR}/cross-linux.cmake")
```
Comment (isuruf) [inline recipes/lhcb-simpletools/build.sh]: Is this necessary?

Comment (chrisburr) [reply recipes/lhcb-simpletools/build.sh]: I'm not sure. I've started including it by default when writing CMake based builds based on the advice [here](https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots).

**Pull request merged**